### PR TITLE
Refactor app run and config for Johnpapa style

### PR DESCRIPTION
**Pull request by deepu105**
Refactor `app.config.js` and `app.module.js` to have run blocks and config blocks as individual files following John papa style for angular

fix https://github.com/jhipster/generator-jhipster/issues/2986

### CODE REVIEW

**Comment by deepu105** [comment]
@wmarques can you look at this, are the changes inline with the style?

**Comment by wmarques** [comment]
Yes the changes seems ok with the style, I refactored the interceptor components in my latest PR #2997 so with those 2 PR it will be fine :+1:

**Comment by deepu105** [comment]
Ok ill merge yours first and then this

File: generators/client/templates/src/main/webapp/app/blocks/config/_alert.config.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('<%=angularAppName%>')
+        .config(alertServiceConfig);
+
+    alertServiceConfig.$inject = ['AlertServiceProvider'];
+
+
```
Comment (mraible) [inline generators/client/templates/src/main/webapp/app/blocks/config/_alert.config.js]: Remove extra line break.

**Comment by deepu105** [comment]
Im merging this as the test failed are not due to this PR its from master

**Pull request merged**